project/BuildPlugin.scala

```diff
@@ -68,6 +72,35 @@ object BuildKeys {
     Keys.libraryDependencies += Dependencies.utest,
     Keys.testFrameworks += new sbt.TestFramework("utest.runner.Framework"),
   )
+
+  import sbtassembly.AssemblyKeys
+  val assemblySettings: Seq[Def.Setting[_]] = List(
```

Comment:
Will assembly directly add the nailgun server and all the bridges to this mega jar?

Reply:
Yes, the nailgun server is part of the assembly, but not the bridges.